### PR TITLE
update readme for (gitlab) config

### PR DESCRIPTION
**Pull request by paulirish**
this is docs for #92

### CODE REVIEW

**Comment by paulirish** [comment]
@derimagia i didn't really expect "https://" to be in the config name, since that isn't part of the actual git repo URL.  is it just a concession to keep the implementation simple? 

if possible seems like it wouldn't be part of the config name. wdyt?

**Comment by derimagia** [comment]
@paulirish I did that so we could take advantage of https://git-scm.com/docs/git-config#git-config---get-urlmatchnameURL (so I think you could even use it to do all subdomains as well?). 

I'm not too attached to it and am fine with changing it - it just looked like a good fit

**Comment by derimagia** [comment]
It looks like it also works with paths, so you can have it so that https://github.com/paulirish goes to one path but https://github.com and any other path goes to another one. 

I'd also mention in the readme that you can do --global. It looks like --local is the default for writing, so I think the way it was before is a little better (`git config [--global]`). Looks like --add isn't needed (I may have had it in the test, but looks like the default is "to replace at most one line").

**Comment by derimagia** [comment]
Had the idea as well that we could take care of a lot of requests with this by also adding a setting for "open.branch" to override the branch like we do with protocol/server

File: README.md [hunk 1]
```diff
@@ -101,27 +101,25 @@ git-open can automatically guess the corresponding repository page for remotes
...
+| open.[gitdomain].protocol | The (web) protocol to open based on the provided git repo domain. (Defaults to `https`) |
 
-```sh
-# use --global to set across all repos, instead of just the local one
-git config [--global] gitopen.gitlab.domain [value]
-git config [--global] gitopen.gitlab.ssh.domain [value]
-git config [--global] gitopen.gitlab.ssh.port [value]
-```
+<!--
+git config --local --add open.[gitdomain].domain [value]
```
Comment (derimagia) [inline README.md]: It looks like --local is the default for writing, so I think the way it was before is a little better (git config [--global]). Looks like --add isn't needed (I may have had it in the test, but looks like the default is "to replace at most one line").

**Comment by paulirish** [comment]
@derimagia 

>  I did that so we could take advantage of git-scm.com/docs/git-config#git-config---get-urlmatchnameURL (so I think you could even use it to do all subdomains as well?).

> I'm not too attached to it and am fine with changing it - it just looked like a good fit

im ok with it, but it's just something we'll have to explain to the user, as its a bit counterintuitive.

do you wanna take a swipe at updating these docs? I think you can speak to it a little better than me. :)

File: README.md (hunk 1 above)
Comment (paulirish) [reply README.md]: ya sgtm on dropping --local and --add. i also was headscratching on what value they added. :)

**Comment by paulirish** [comment]
Alright I made the basic change here of `--local --add` ==> `[--global]`.

I left the https:// prefix the same. I think it probably need some additional explanation at some point, but I don't want to hold up the docs on it right now.

> It looks like it also works with paths

This is also something we could document if it comes up.

> Had the idea as well that we could take care of a lot of requests with this by also adding a setting for "open.branch" to override the branch like we do with protocol/server

Seems interesting. If anyone is fond of the idea plz feel free to open an issue so we can explore it a bit more.

**Pull request merged**